samples/WebApi/SampleWebApi.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+        <RdXmlFile>rd.xml</RdXmlFile>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <IlcArg Include="--noscan" />
```

Comment:
Adding the "--noscan" compiler option should not be necessary after https://github.com/dotnet/corert/pull/5040 is merged. I would remove it from the sample.